clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -228,19 +249,17 @@ const TcfOverlay: FunctionComponent<OverlayProps> = ({
             onClose={onClose}
             experience={experienceConfig}
           >
-            <InitialLayer
-              experience={experience}
-              managePreferencesLink={
-                <button
-                  type="button"
-                  onClick={onManagePreferencesClick}
-                  className="fides-link-button"
-                >
-                  {experience.experience_config
-                    ?.privacy_preferences_link_label || ""}
-                </button>
-              }
-            />
+            <InitialLayer experience={experience} />
+            <button
+              type="button"
+              className="fides-link-button"
+              onClick={() => {
+                onManagePreferencesClick();
+                goToVendorTab();
+              }}
+            >
+              View our {numVendors} partner{numVendors > 1 ? "s" : ""}
```

Comment:
will this button show if there are 0 systems + vendors?

Reply:
it will! but in theory it shouldn't be possible to have 0 vendors since then no purposes/features would show up. but also I asked Rachel and she said she's ok with it saying 0 :) I suppose in that case it should say 0 vendor`s`...